test/refactoring/ConvertAsync/basic.swift

```diff
@@ -151,14 +151,9 @@ func retStruct() -> MyStruct { return MyStruct() }
 
 protocol MyProtocol {
   // RUN: %refactor -add-async-alternative -dump-text -source-filename %s -pos=%(line+2):3 | %FileCheck -check-prefix=PROTO-MEMBER %s
-  // RUN: %refactor -convert-to-async -dump-text -source-filename %s -pos=%(line+1):3 | %FileCheck -check-prefix=PROTO-MEMBER-TO-ASYNC %s
+  // RUN: %refactor -convert-to-async -dump-text -source-filename %s -pos=%(line+1):3 | %FileCheck -check-prefix=PROTO-MEMBER %s
```

Comment:
Is there a test that checks that it does actually *convert* and not *add* anywhere? See the top of this file for an example (it's for add-async, convert should have an actual range rather than a 0 length one).

Reply:
I don’t think so. I’ll add one in a follow-up PR.